### DISCUSSION TITLE
Welcome to rpi-kvm Discussions!

### DISCUSSION THREAD

**Discussion by BLeeEZ** [BLeeEZ/rpi-kvm#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by argenkiwi** [comment]
Hi there. I work as an Android Developer in New Zealand and I am interested in, among other things, keyboard customisation. I have been looking for ways to help myself and others get the benefits of custom keyboard layers, key overloads, etc., without having to spend a lot of money on fancy hardware. I recently published a [post](https://www.reddit.com/r/KeyboardLayouts/comments/1hhawfi/make_your_keyboard_programmable_with_a_raspberry/) on how rpi-kvm can be used for that purpose. Thanks for this project and keep up the good work!
